modules/transactionpool/accept.go

```diff
@@ -331,18 +331,33 @@ func (tp *TransactionPool) AcceptTransactionSet(ts []types.Transaction) error {
 		return errors.New("consensus set does not support LockedTryTransactionSet method")
 	}
 
-	return cs.LockedTryTransactionSet(func(txnFn func(txns []types.Transaction) (modules.ConsensusChange, error)) error {
+	err := cs.LockedTryTransactionSet(func(txnFn func(txns []types.Transaction) (modules.ConsensusChange, error)) error {
 		tp.mu.Lock()
 		defer tp.mu.Unlock()
 		err := tp.acceptTransactionSet(ts, txnFn)
 		if err != nil {
```

Comment:
`if err == nil`, right now you are only updating subscribers if there is an error :P